File: Linguagem C++/Matrizes/NegativosMatriz.cpp
#include <iostream>

using namespace std;

int main()
{
    int m,n;

    cout << "Qual a quantidade de linhas da matriz? ";
    cin >> m;
    cout << "Qual a quantidade de colunas da matriz? ";
    cin >> n;

    int mat[m][n];

    /// Lendo a matriz
    for (int i=0;i<=m-1;i++){
        for (int j=0;j<=n-1;j++){
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> mat[i][j];
        };
    };

    /// Imprimindo os valores negativos
    cout << "Valores Negativos:" << endl;
    for (int i=0;i<=m-1;i++){
        for (int j=0;j<=n-1;j++){
            if (mat[i][j] < 0){
                cout << mat[i][j] << endl;
            };
        };
    };

    return 0;
}

File: Linguagem C#/Matrizes/NegativosMatriz.cs
using System;

namespace NegativosMatriz {
    class Program {
        static void Main(string[] args) {

            int m, n;

            Console.Write("Qual a quantidade de linhas da matriz? ");
            m = int.Parse(Console.ReadLine());
            Console.Write("Qual a quantidade de colunas da matriz? ");
            n = int.Parse(Console.ReadLine());

            int[,] mat = new int[m, n];

            for (int i = 0; i < m; i++) {
                for(int j = 0; j < n; j++) {
                    Console.Write("Elemento ["+i+","+j+"]: ");
                    mat[i, j] = int.Parse(Console.ReadLine());
                };
            };

            Console.WriteLine("VALORES NEGATIVOS:");
            for (int i = 0; i < m; i++) {
                for (int j = 0; j < n; j++) {
                    if (mat[i,j] < 0) {
                        Console.WriteLine(mat[i,j]);
                    };
                };
            };
        }
    }
}
